packages/pages/src/vite-plugin/modules/plugin.ts

```diff
@@ -92,7 +99,7 @@ export const buildModules = async (
         },
       },
       experimental: {
-        renderBuiltUrl(filename, { type }) {
+        renderBuiltUrl(filename: any, { type }: any) {
```

Comment:
Can you give this the real types? `any` is a catchall and you generally don't want to use it if possible

Reply:
double checked types w/ vite's documentation